addons/new-osio-addon/src/test/java/io/fabric8/launcher/osio/steps/OpenShiftStepsTest.java

```diff
@@ -28,13 +32,15 @@
 import org.junit.contrib.java.lang.system.ProvideSystemProperty;
 import org.junit.rules.RuleChain;
 
-import static io.fabric8.launcher.osio.HoverflyRuleConfigurer.createHoverflyProxy;
-
 public class OpenShiftStepsTest {
 
     @ClassRule
     public static RuleChain ruleChain = RuleChain
-            .outerRule(new ProvideSystemProperty("https.proxyHost", "127.0.0.1")
+            // After recording on a real environment against a real service,
+            // You should adapt the Hoverfly descriptors (.json) to make them work in simulation mode with the mock environment.
```

Comment:
Ok, this is much better. Is there a document somewhere that tells you _how_ to do this? ANd I don't mean the hoverfly docs but what changes we need to make in our particular case?